index_single.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
     <title>Open Errata for the JSON-LD API specification published by the JSON-LD Working Group</title>
     <link rel="stylesheet" type="text/css" href="errata.css"/>
-    <script src="https://w3c.github.io/display_errata/assets/errata.js" type="text/javascript"></script>
+    <script src="https://w3c.github.io/display_errata/ass/exploreets/errata.js" type="text/javascript"></script>
```

Comment:
this one doesn't make sense to me. it doesn't look like you actually moved the CSS file...